org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -53,6 +53,8 @@ define([
             return error('custom settings');
           }
 
+          customSettings.allowedMimeTypes = customSettings.allowedMimeTypes || '|plain|png|jpeg|bmp|gif|pdf|';
```

Comment:
write a private function where you set the default of `customSettings`, and you use `_.defaults`